docs/deployment-environment.mdx

```diff
@@ -23,7 +23,12 @@ If the new deployment fails to become healthy, it will be retried 3 times while
 
 HTTPS traffic is proxied to your app on the project's default subdomain and any [custom domains]() that have been added.
 
-The proxy sets the `X-Forwarded-For` HTTP header on incoming requests to the remote IP address of the request.
+### Request Headers
+
+#### X-Forwarded-For
+`X-Forwarded-For` is a header that allows you to get the original IP of the HTTP request. Deployed Shuttle projects sit behind a proxy, meaning that when you try to get the IP address normally you may end up with the proxy IP instead of the requester IP. With the `X-Forwarded-For` header, you can now always get the requester IP by parsing the given header value.
```

Comment:
Should we make it a bit more concise, perhaps like this?
```suggestion
Deployed Shuttle projects sit behind a proxy, so if you extract the peer address of the incoming connection in your application, it will be the address of our proxy. To allow you to get the IP address of the original client, the Shuttle proxy will set it in the `X-Forwarded-For` (link to the mdn docs for the header here) header in all requests to your project. 
```

We are also using the header incorrectly, it should be a comma separated list of IPs, including the IP of our proxy. If it's just the original client IP, we should have a new custom header, as I understand it. But we can address that later.